### PR TITLE
#163901934 Ensure no duplicate entries and valid status codes

### PR DESCRIPTION
**Pull request by luqee**
#### What does this PR do?
Ensure no duplicate entries and valid status codes.
#### Description of Task to be completed?
Ensure that when an office or party is created, there should be no duplicates and the returned status codes should be valid.
#### What are the relevant pivotal tracker stories?
[163901934](https://www.pivotaltracker.com/story/show/163901934)
####Screenshots
party exists
office exists

### CODE REVIEW

File: app/api/v1/blueprints/offices.py [hunk 1]
```diff
@@ -87,25 +93,33 @@ def update_office(office_id):
...
\ No newline at end of file
+        return jsonify(response), 202
+    elif party == 'Party not found':
+        response = {
+            'status': 404,
+            'data':[{
+                'error': 'Party not found'
+            }]
+        }
+        return jsonify(response, 404)
```
Comment (meshack-mbuvi) [inline app/api/v1/blueprints/offices.py]: Add a new line after at the end of the file

File: app/tests/v1/test_parties.py [hunk 2]
```diff
@@ -60,8 +60,9 @@ def test_delete_party(client):
         'Authorization': 'Bearer {0}'.format(login_res.get_json()['data'][0]['auth_token'])
     }
     client.post('api/v1/parties', json=test_utils.PARTIES[0], headers=headers)
-    response =client.delete('api/v1/parties/1', headers=headers)
-    json_data = response.get_json()
-    assert response.status_code == 200
+    req_response =client.delete('api/v1/parties/1', headers=headers)
+    print(req_response)
```
Comment (meshack-mbuvi) [inline app/tests/v1/test_parties.py]: Remove print statements

File: app/api/v1/blueprints/parties.py [hunk 3]
```diff
@@ -30,10 +30,16 @@ def create_party():
         return jsonify(response), 201
     elif result == 'Not authorised':
```
Comment (meshack-mbuvi) [inline app/api/v1/blueprints/parties.py]: the value of result on line 31 should not be **Not authorised**

File: app/tests/v1/test_parties.py [hunk 4]
```diff
@@ -60,8 +60,9 @@ def test_delete_party(client):
...
+    json_data = req_response.get_json()
+    assert req_response.status_code == 202
     assert type(json_data['data']) == list
     assert json_data['data'][0]['message'] == 'Party deleted successfully'
```
Comment (meshack-mbuvi) [inline app/tests/v1/test_parties.py]: Blank line at the end of every file.

**Review by meshack-mbuvi — CHANGES_REQUESTED** [review]

File: app/api/v1/blueprints/offices.py (hunk 1 above)
Comment (luqee) [reply app/api/v1/blueprints/offices.py]: Implemented

File: app/tests/v1/test_parties.py (hunk 2 above)
Comment (luqee) [reply app/tests/v1/test_parties.py]: Print statements removed

File: app/api/v1/blueprints/parties.py (hunk 3 above)
Comment (luqee) [reply app/api/v1/blueprints/parties.py]: Rectified

File: app/tests/v1/test_parties.py (hunk 4 above)
Comment (luqee) [reply app/tests/v1/test_parties.py]: Changed.

**Review by meshack-mbuvi — APPROVED** [review]

**Pull request merged**